src/logic/BrazeMessages.tsx

```diff
@@ -123,13 +159,17 @@ class BrazeMessages implements BrazeMessagesInterface {
 
     getMessageForBanner(): Promise<BrazeMessage> {
         // If there's already a message in the cache, return it
-        const messageFromCache = this.cache.peek('Banner', this.appboy, this.errorHandler);
```

Comment:
Just noting that we've only made these changes to `getMessageForBanner` but not `getMessageForEndOfArticle`. That's totally fine if it's what we meant to do, but wanted to mention. What do you reckon, should we change both?

Reply:
This is fixed now :)